# Homework 1

choose the best option:
1. $1000 now
2. $2000 at the end of next 3 years
3. $3000 after 10 years

- Case1: discount rate = `10%`
- Case2: discount rate = `5%`

### Case 1 (r = 0.1)

In [26]:
r = 0.1

In [27]:
# PV = FV/((1+r)**t)
FV = 2000
t = 3

PV = FV/(1+r)**t
PV

1502.629601803155

In [28]:
# PV = FV/((1+r)**t)
FV = 3000
t = 10

PV = FV/(1+r)**t
PV

1156.6298682885943

Since

- Choice 1: PV = 1000
- Choice 2: PV = `1502.63`
- Choice 3: PV = 1156.63

So `Choice 2` is worth the most 

### Case 2 (r = 0.05)

In [29]:
r = 0.05

In [30]:
# PV = FV/((1+r)**t)
FV = 2000
t = 3

PV = FV/(1+r)**t
PV

1727.675197062952

In [31]:
# PV = FV/((1+r)**t)
FV = 3000
t = 10

PV = FV/(1+r)**t
PV

1841.7397606222773

Since

- Choice 1: PV = 1000
- Choice 2: PV = 1727.68
- Choice 3: PV = `1841.74`

So `Choice 3` is worth the most 